src/HttpRouter.h

```diff
@@ -257,27 +221,21 @@ class HttpRouter {
      * In that case, a second pass where method changed to "*" to denote "any" could be used to
      * give such routes a chance. If second pass fails, we have an unhandled request and you may
      * do whatever you want with your connection, such as close it, or respond with a fix message */
-    bool route(std::string_view method, std::string_view url, USERDATA &userData) {
+    bool route(const std::string_view method, const std::string_view url, USERDATA &userData) {
         /* Reset url parsing cache */
         setUrl(url);
         routeParameters.reset();
 
         /* Begin by finding the method node */
-        Node *parent = &tree;
-        for (auto &p : parent->children) {
-            if (p->name == method) {
-                parent = p;
-            }
-        }
-
-        /* We have that method on record, let's iterate it */
-        if (parent != &tree) {
-            return executeHandlers(parent, 0, userData);
-        } else {
-            /* We did not find any handler for this method.
-             * You may want to re-route with "*" as method. */
-            return false;
-        }
+		for (auto &p : tree.children) {
```

Comment:
This is basically

```
for (auto &c : tree->children) {
	if (c->name == method) {
		return executeHandlers(c.get(), 0, userData);
	}
}
return false;
```

Reply:
Yes. Have no idea why github shows changes this way.
This is the file
https://github.com/guffy1234/uWebSockets/blob/master/src/HttpRouter.h